dagobah/daemon/api.py

```diff
@@ -32,7 +32,7 @@ def get_job():
     if not job:
         abort(400)
 
-    return job._serialize()
+    return job._serialize(use_snapshot='use_snapshot' in args)
```

Comment:
Did you intend for use_snapshot to be true as long as it in set, regardless of the value?